### PR TITLE
frontend statistics layout

### PR DESCRIPTION
**Pull request by yoouung**
- implement statistics page layout
- add bar charts, pie charts
- TODO: reflect data from server after api reqeust impelemented
- TODO: match the pie charts' position if possible

### CODE REVIEW

**Comment by yoouung** [comment]
### Statistics page looks like:

image

**Comment by doorooful** [comment]
Can you add sample API returning for the further implementation?

File: dt-demo-be/app_lab.py [hunk 1]
```diff
@@ -129,5 +129,25 @@ def auto_get_conv_report():
...
+# ------------ statistics API ------------
+
+@app.route("/api/statistics", methods=["GET"])
+def get_statistics_data():
+    app.logger.info("API request to get statistics data")
+
+    # TODO: remove me, or use only for developing purpose
+    data = {
+        "average_num_per_module":{"typo":46,"slang":35,"dup":25,"pdd":5,"spc":26},
+        "cumulated_num_per_module":{"typo":267,"slang":162,"dup":85,"pdd":78,"spc":200},"typo_proportion":{"noun":10,"verb":20,"josa":30,"etc":40},
```
Comment (doorooful) [inline dt-demo-be/app_lab.py:143]: Can you add line break after comma?

File: dt-demo-fe/src/components/chart/data.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        },
+    ],
+};
+
+export const cumData = {
+    labels: labels,
+    datasets: [
+        {
+            data: { typo: null, slang: null, dup: null, pdd: null, spc: null },
+            backgroundColor: ["#ffc0cb", "#eee8d1", "#d2eddd", "#c3e6fc", "#c2d3fd"],
```
Comment (doorooful) [inline dt-demo-fe/src/components/chart/data.js:50]: Can you define this as a static variable and file to refer?

File: dt-demo-fe/src/components/chart/data.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            borderWidth: 1,
+        },
+    ],
+};
+
+// ====== pie chart data =======
+
+export const modulesProportionData = {
+    typoData: {
+        labels: ["명사", "동사", "조사", "기타"],
```
Comment (doorooful) [inline dt-demo-fe/src/components/chart/data.js:60]: Please note types of labels as to be discussed

File: dt-demo-fe/src/components/chart/data.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            {
+                data: { noun: null, verb: null, josa: null, etc: null },
+                backgroundColor: ["#e0677c", "#e68798", "#ed9dab", "#ebc5cc"],
+                borderWidth: 1,
+            },
+        ],
+    },
+
+    slangData: {
+        labels: ["욕설 및 음란어", "혐오", "폭력 및 협박", "기타"],
```
Comment (doorooful) [inline dt-demo-fe/src/components/chart/data.js:71]: Same here

**Review by doorooful — APPROVED** [review]
Please find the comments😊

**Pull request merged**